docs/user/admin.rst

```diff
@@ -0,0 +1,204 @@
+Admin
+===============
+
+Installing Kolibri
+------------------
+
+Windows
+~~~~~~~~
+
+To install Kolibri on Windows follow these steps:
+
+#. Download the Kolibri Windows installer. **(TODO - link)**
+#. Double-click the downloaded ``.exe`` file.
+#. Follow the instructions in the installation wizard window.
+
+
+Creating a Device Owner Account
+-------------------------------
+
+In order to manage Kolibri content channels, data and users, you must first create your own **Device Owner** account and **Facility**. The **Device Owner** account registration page appears when you open the browser at http://127.0.0.1:8000/ after the installation of Kolibri.
+
+.. image:: img/wizard.png
+  :alt: create device owner form
+
+#. Enter the required information for your **Device Owner** account (username, password, name of the facility). Fields marked with an asterisk (*) are required.
+#. Select **Create and get started**.
+
+
+Adding a New Device Owner
```

Comment:
This section is going to be optional for most people. I wonder if it would make sense to push it to the bottom of the page under the title "Need to make a new Device Owner?" or something?